app/src/main/java/org/p2p/wallet/rpc/RpcConstants.kt

```diff
@@ -0,0 +1,10 @@
+package org.p2p.wallet.rpc
+
+object RpcConstants {
+
+    const val REQUEST_METHOD_KEY = "method"
+    const val GET_CONFIRMED_TRANSACTIONS = "getConfirmedTransaction"
```

Comment:
`GET_CONFIRMED_TRANSACTIONS` -> `REQUEST_METHOD_VALUE_GET_CONFIRMED_TRANSACTIONS` 
like you did with REQUEST_PARAMETER